cmd/sourced/compose/workdir/common.go

```diff
@@ -327,6 +339,24 @@ func initWorkdir(workdirPath string, envFile envFile) error {
 		}
 	}
 
+	defaultOverridePath, err := composefile.InitDefaultOverride(composePath)
+	if err != nil {
+		return err
+	}
+
+	workdirOverridePath := filepath.Join(workdirPath, "docker-compose.override.yml")
+	_, err = os.Stat(workdirOverridePath)
```

Comment:
this is copy-past. let's create helper function `symlink` and use it here and for `docker-compose.yml`.

Reply:
good idea, done by 6e4ca6a0c4367706f4b6d1af64086b6992d3d1e1